pkgs/development/python-modules/tensorboardx/default.nix

```diff
@@ -63,6 +63,8 @@ buildPythonPackage rec {
   disabledTestPaths = [
     # we are not interested in linting errors
     "tests/test_lint.py"
+    # breaks with `RuntimeError: cannot schedule new futures after interpreter shutdown`
```

Comment:
We should link to an upstream issue for this, or create one if it has not already been reported.

Reply:
Created https://github.com/lanpa/tensorboardX/issues/652 and linked here.